Classes/Hooks/CachingFrameworkLoggerHook.php

```diff
@@ -68,20 +68,16 @@ public function analyze(array $parameters, VariableFrontend $frontend): void
                 $cacheExpressionRecord,
             );
 
-            if ($cacheExpressionRecord['is_exception']) {
-                // throw exception in FE only or also in BE, if fe_only is disabled
-                if (
-                    $cacheExpressionRecord['exception_fe_only'] === 0
-                    || (
-                        $cacheExpressionRecord['exception_fe_only'] === 1
-                        && ApplicationType::fromRequest($this->getServerRequest())->isFrontend()
-                    )
-                ) {
-                    throw new PreventStoringFalseCacheEntryException(
-                        '[jwtools2] CF logger prevents inserting invalid cache entry',
-                        1720607181,
-                    );
-                }
+            // throw exception in FE only or also in BE, if fe_only is disabled
+            if ($cacheExpressionRecord['is_exception'] && ($cacheExpressionRecord['exception_fe_only'] === 0
+            || (
+                $cacheExpressionRecord['exception_fe_only'] === 1
+                && ApplicationType::fromRequest($this->getServerRequest())->isFrontend()
+            ))) {
```

Comment:
Update to, or move into own method for better reading:

```suggestion
            // throw exception in FE only or also in BE, if fe_only is disabled
            if ($cacheExpressionRecord['is_exception']
                && (
                    $cacheExpressionRecord['exception_fe_only'] === 0
                    || (
                        $cacheExpressionRecord['exception_fe_only'] === 1
                        && ApplicationType::fromRequest($this->getServerRequest())->isFrontend()
                    )
                )
            ) {
```